modular_skyrat/modules/customization/modules/language/arabic.dm

```diff
@@ -0,0 +1,13 @@
+/datum/language/arabic
+	name = "Xerxian"
+	desc = "An evolved form of the language of Arabic from Earth. Despite its ancient origins, it is still spoken by many people in cultures that evolved from the Middle-East."
+	key = "a"
+	flags = TONGUELESS_SPEECH
+	syllables = list(
+		"a", "ah", "ba", "lah", "tha", "shah", "sa", "cha", "zah", "in", "al", "an",
+		"i", "ih", "bi", "lih", "thih", "shih", "sih", "chi", "zih", "ji", "nn", "jo", "ja",
+        "o", "oh", "boh", "loh", "tho", "sho", "so", "cho", "zo", "ya", "yi", "yo"
+	)
+	icon_state = "arabic"
```

Comment:
again, probably best the icon is closer to the languages name in-game instead of what you based it off of

Also, on this icon, is there any way to make the symbols on the flag take up more space without ruining the sprite? It feels really empty and off-center as it is right now, but that may be intentional idk

Reply:
fixed the icon name, i think the flag is fine because the little emblem on it was meant to be pretty small